discord/gameStateMessage.go

```diff
@@ -10,19 +10,45 @@ type GameStateMessage struct {
 	lock    sync.RWMutex
 }
 
+type PrivateStateMessage struct {
+	message *discordgo.Message
+	lock sync.RWMutex
+	idUsernameMap map[string]string
+	printedUsers []string
+	privateChannelID string
+	currentUserID string
+}
+
 func MakeGameStateMessage() GameStateMessage {
 	return GameStateMessage{
 		message: nil,
 		lock:    sync.RWMutex{},
 	}
 }
 
+func MakePrivateStateMessage() PrivateStateMessage {
+	return PrivateStateMessage{
+		message: nil,
+		lock:    sync.RWMutex{},
+		idUsernameMap: make(map[string]string),
+		printedUsers: make([]string, 0),
+		privateChannelID: "000000000000000000", // Default Value
```

Comment:
Not sure I would recommend this as a default? An empty string is easier to check for being an idiomatic "null" value (we should never have an empty string for a channelID in any legitimate usage scenario)

Reply:
Ah yea that makes more sense. We'd expect it to be something resembling null/empty like an empty string rather than a non-null/empty. Thank you! Should be changed with the recent update.